addons/event_sale/models/sale_order.py

```diff
@@ -67,3 +69,28 @@ def _get_product_catalog_domain(self):
         """
         domain = super()._get_product_catalog_domain()
         return expression.AND([domain, [('service_tracking', '!=', 'event')]])
+
+    def _notify_get_recipients_groups(self, message, model_description, msg_vals=None):
+        groups = super()._notify_get_recipients_groups(message, model_description, msg_vals)
+        if not self or self.state != 'sale' or not self.order_line.registration_ids:
+            return groups
+
+        try:
+            customer_portal_group = next((group for group in groups if group[0] == 'portal_customer'), None)
+        except StopIteration:
+            return groups
+
+        if customer_portal_group[2]['has_button_access']:
```

Comment:
Was wondering about this access, but looks like the super will always have proper structure :+1: